cvmfs/cvmfs_talk.cc

```diff
@@ -254,14 +255,23 @@ int main(int argc, char *argv[]) {
     std::string opt_repos;
     options_mgr.GetValue("CVMFS_REPOSITORIES", &opt_repos);
     std::vector<std::string> repos = SplitString(opt_repos, ',');
+    bool is_empty_repo_list = true;
     for (unsigned i = 0; i < repos.size(); ++i) {
       if (repos[i].empty())
         continue;
+      is_empty_repo_list = false;
       instance_info.instance_name = repos[i];
       LogCvmfs(kLogCvmfs, kLogStdout, "%s:", repos[i].c_str());
       bool retval = SendCommand(command, instance_info);
       if (!retval) retcode = 1;
     }
+    if (is_empty_repo_list) {
+      LogCvmfs(kLogCvmfs, kLogStdout,
+               "Warning: no instance was specified. In this case, the command "
+               "is executed for all instances defined in CVMFS_REPOSITORES but "
+               "this list is empty. Did you mean running \n\n"
+               "    cvmfs_talk -i <repository name> <command>\n");
```

Comment:
do you also want to add the option with `-p <socket>`?

Reply:
I think I'd rather not advertise the `-p` interface. Under normal circumstances, `-i` should be used.